### PR TITLE
Move Kubelet pod-management code into kubelet_pods.go

### PR DESCRIPTION
**Pull request by pmorie**
Finish the kubelet code moves started during the 1.3 dev cycle -- move pod management code into a file called `kubelet_pods.go`.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33432)

### CODE REVIEW

**Comment by pmorie** [comment]
@k8s-bot gke e2e test this

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 The Kubernetes Authors.
+Copyright 2016 The Kubernetes Authors.
```
Comment (derekwaynecarr) [inline pkg/kubelet/kubelet.go]: why does this change?

File: pkg/kubelet/kubelet.go [hunk 2]
```diff
@@ -1248,423 +1218,6 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
...
-
-	opts.DNS, opts.DNSSearch, err = kl.GetClusterDNS(pod)
-	if err != nil {
-		return nil, err
-	}
-
-	return opts, nil
-}
-
-var masterServices = sets.NewString("kubernetes")
```
Comment (derekwaynecarr) [inline pkg/kubelet/kubelet.go]: need to work down why this moved....

Comment (derekwaynecarr) [inline pkg/kubelet/kubelet.go]: ignore above, now i see...

**Review by derekwaynecarr — CHANGES_REQUESTED** [review]
just the one nit/question.

i do not look forward to rebases.

File: pkg/kubelet/kubelet.go (hunk 1 above)
Comment (pmorie) [inline pkg/kubelet/kubelet.go]: Why shouldn't it?

**Comment by pmorie** [comment]
@derekwaynecarr nit addressed

File: pkg/kubelet/kubelet.go (hunk 2 above)
Comment (pmorie) [inline pkg/kubelet/kubelet.go]: Consider it ignored

**Comment by pmorie** [comment]
@k8s-bot kubemark gci e2e test this

**Review by derekwaynecarr — APPROVED** [review]

**Pull request merged**